content/blog/2021-07-27-july-21-dvc-community-gems.md

```diff
@@ -0,0 +1,131 @@
+---
+title: July '21 Community Gems
+date: 2021-07-27
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: self-hosted runners, DVC commits,
+  troubleshooting remotes, and more.
+picture: 2021-07-27/gems-cover.png
```

Comment:
image is not being rendered I think ... are we waiting on it still?

Reply:
Yep! We're still waiting on this.

Reply:
Is this the only thing missing? Do you guys have the ability to merge? @flippedcoder @jendefig otherwise please tag us when ready 🙂

Reply:
@flippedcoder Are you going to put the pic in or do you need me to?